testController.js

```diff
@@ -1,8 +1,37 @@
 document.addEventListener('DOMContentLoaded', function() {
-    document.getElementById('home').addEventListener('click', function() {
-        console.log('vleze')
-        getWeather('sofia').then(function(data) {
-            console.log(data)
+        function getCity() {
+            return document.getElementById('city').value
+        }
+    document.getElementById('enterCity').addEventListener('click', function() {
```

Comment:
a good practice is to define named functions and provide them as callbacks, because in this way you can removeEventListener on a later stage and prevent memory leaks.